pkg/tagger/collectors/ecs_fargate_extract_test.go

```diff
@@ -89,7 +89,7 @@ func TestParseMetadata(t *testing.T) {
 		},
 		{
 			Source: "ecs_fargate",
-			Entity: "docker://0fc5bb7a1b29adc30997eabae1415a98fe85591eb7432c23349703a53aa43280",
+			Entity: "container_id://0fc5bb7a1b29adc30997eabae1415a98fe85591eb7432c23349703a53aa43280",
```

Comment:
I see `docker` was replaced everywhere, but I don't see `kubernetes://` being replaced. I think this package is a bit out of my area of expertise. Happy to try it out as soon as it gets merged. Or, if you wanna try it out from the branch, we can do that too (it's code freeze soon).

Reply:
we don't replace `kubernetes`, but `kubernetes_pod://`